pkg/limits/store.go

```diff
@@ -226,10 +231,16 @@ func (s *usageStore) UpdateCond(tenant string, metadata []*proto.StreamMetadata,
 				}
 			}
 			s.update(i, tenant, partition, m, seenAt)
+			// Hard-coded produce cutoff of 1 minute.
+			produceCutoff := now.Add(-time.Minute).UnixNano()
+			if stream.lastProducedAt < produceCutoff {
```

Comment:
When the stream is new, `lastProducedAt` is `0`, which is always less than `produceCutoff`, ensuring new streams are always produced.